src/common-ui/Toggle.tsx

```diff
@@ -0,0 +1,120 @@
+import { theme } from '@/styles/theme';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import Text from './Text';
+
+interface Props {
+  firstText: string;
+  secondText: string;
+  isToggle: boolean;
+  setToggleTrue: () => void;
+  setToggleFalse: () => void;
```

Comment:
아래와 같이 이벤트 핸들러 하나로 상태값 받을 수 있도록 하는거 에 대해서는 어떻게 생각하시나용? 그럼 외부에서도 이벤트 핸들러 하나만 붙여서 처리할 수 있어서 깔끔할 거 같습니당

```suggestion
  onChangeToggle: (isToggle:boolean)=>void;
```

Reply:
요거는 고민을 했었는데 각 상태에 따라서 콜백 내부에 바인딩 하고 싶은 다른 조건들을 
`onChangeToggle` 하나로 묶을 시 안에서 분기 처리를 해야할 것 같아서 🧐
하나로 처리해 버릴까요?